draft-demarco-oauth-status-attestations.md

```diff
@@ -464,6 +467,8 @@ The Wallet Instance that provides the Status Attestations using [@OpenID4VP], SH
 `vp_token` JSON array, as defined in [@OpenID4VP], the Status Attestation along with the
 related Digital Credential.
 
+Since the Wallet may request one or more Status Attestations, issued by the same Credential Issuer, the `credential_pop` object MUST be an array.
```

Comment:
this is a consideration that is not required here, this must be merged within the definition of `credential_pop`